rebar.config

```diff
@@ -35,9 +37,22 @@
 {erl_opts, [debug_info,
 	    report_warnings, {warn_format,1}, warn_export_vars,
 	    warn_obsolete_guard, warn_unused_import,
-	    warn_missing_spec, warn_untyped_record]}.
+	    warn_missing_spec, warn_untyped_record,
+	    warn_unused_vars,
```

Comment:
I thought that `warn_unused_vars` was on by default...  Do you have evidence to the contrary?